engine/src/main/java/org/terasology/engine/rendering/dag/AbstractNode.java

```diff
@@ -472,7 +474,7 @@ public List<String> getOutputConnectionsList() {
     }
 
     public void disconnectInputFbo(int inputId) {
-        logger.info("Disconnecting {} input Fbo number {}", this.getUri(), inputId);
+        logger.atInfo().addArgument(() -> this.getUri()).addArgument(inputId).log("Disconnecting {} input Fbo number {}");
```

Comment:
simple getter method
```suggestion
        logger.info("Disconnecting {} input Fbo number {}", this.getUri(), inputId); //NOPMD
```